src/CustomButtons.js

```diff
@@ -0,0 +1,8 @@
+export function CustomButtons({btnID}){
```

Comment:
Format the destructuring
```suggestion
export function CustomButtons({ btnID }){
```